matter/efr32/efr32mg24/BRD2601B/autogen/sl_ot_custom_cli.c

```diff
@@ -57,5 +57,7 @@ const uint8_t sl_ot_custom_commands_count = OT_ARRAY_LENGTH(sl_ot_custom_command
 
 void sl_ot_custom_cli_init(void)
 {
+#if defined(CHIP_DEVICE_CONFIG_THREAD_ENABLE_CLI) && CHIP_DEVICE_CONFIG_THREAD_ENABLE_CLI
```

Comment:
Will this still be necessary after the OT patch? If not, might need a comment, otherwise might be worth opening an OT issue to let them know their patch isn't working.